.github/workflows/basic-checks.yml

```diff
@@ -1,6 +1,6 @@
-name: Vet and Test
+name: GO get and vet
 
-on: [push, pull_request]
+on: [push]
```

Comment:
Is there a reason we're not running this on pull request anymore?

Reply:
When I opened a PR the same action is run twice and there's no need for that
If we run it on push it runs whether it's a commit into a branch or into an open PR. Making a PR out of a branch doesn't mean we have to rerun action again, there's no point - the code won't change with a PR.

Reply:
Ah I thought it was required to be ran on a pull request for the required statuses to work, but that might not be the case